src/components/Text/style.js

```diff
@@ -10,7 +10,7 @@ import {
 } from '../style/fonts';
 
 export const Span = styled.span`
-  color: inherit;
+  color: aquamarine;
```

Comment:
heya @msanroman did this maybe stick around unintended?

Reply:
Hehe, it was just the test! Will remove all silly commits before merging the script 😊